scripts/helper/docker-generate-certificates.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -e
+
+# Load env file
+set -a
+test -f $(dirname $0)/../../containers/.env && source $(dirname $0)/../../containers/.env
```

Comment:
It is not specified in the `README.md` that we need to copy the .env file and without it it's not working by default. We should maybe warn the user if we don't find some env variables and stop the script with an explicit message.

Reply:
It does that now. Plus I added some section to the README